gobblin-restli/server.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
   }
 
   compile externalDependency.gson
+  compile externalDependency.lombok
```

Comment:
unclear how this was missed... but not having meant MANY compilation warnings

Reply:
just curious, how did you find that this was missing